pyproject.toml

```diff
@@ -1,3 +1,3 @@
 [tool.black]
 line-length = 88
-target-version = ['py36', 'py37', 'py38']
+target-version = ['py36', 'py37', 'py38', 'py310']
```

Comment:
suggestion (non-blocking): it'd be good to drop support for python 3.6, since it's EOL.

not a blocker to merging, though.

Reply:
ok. I've changed version's list (and rechecked via tox)
```
target-version = ['py37', 'py38', 'py39', 'py310', 'py311']
```